Installation.md

```diff
@@ -8,15 +8,12 @@ To install Swift for TensorFlow, download one of the packages below and follow t
 
 ## Pre-built Packages
 
-Packages will be released nightly after automated building is set up.
-
-| Download | Kind | Version |
-|----------|------|---------|
-| [Xcode 10](https://storage.googleapis.com/swift-tensorflow/mac/swift-tensorflow-RELEASE-0.2-osx.pkg) | Release | 0.2 |
-| [Xcode 10](https://storage.googleapis.com/swift-tensorflow/mac/swift-tensorflow-DEVELOPMENT-2019-02-25-a-osx.pkg) | Development Snapshot | February 25, 2019 |
-| [Ubuntu 18.04 (CPU Only)](https://storage.googleapis.com/s4tf-kokoro-artifact-testing/latest/swift-tensorflow-DEVELOPMENT-ubuntu18.04.tar.gz) | Development Snapshot | Nightly Build |
-| [Ubuntu 18.04 (CUDA 10.0)](https://storage.googleapis.com/s4tf-kokoro-artifact-testing/latest/swift-tensorflow-DEVELOPMENT-cuda10.0-cudnn7-ubuntu18.04.tar.gz) | Development Snapshot | Nightly Build |
-| [Ubuntu 18.04 (CUDA 9.2)](https://storage.googleapis.com/s4tf-kokoro-artifact-testing/latest/swift-tensorflow-DEVELOPMENT-cuda9.2-cudnn7-ubuntu18.04.tar.gz) | Development Snapshot | Nightly Build |
+| Download | Version | Date | 
+|----------|---------|------|
+| [Xcode 10](https://storage.googleapis.com/swift-tensorflow/mac/swift-tensorflow-DEVELOPMENT-2019-02-25-a-osx.pkg) | v0.2 | February 25, 2019 |
```

Comment:
This binary and date are not the release version. The release is here: https://github.com/tensorflow/swift/blob/master/Installation.md.

Reply:
Oops. Thanks for catching this. Fixed now. (Also changed the date to match linux release date.)